src/components/header/notifications.js

```diff
@@ -0,0 +1,109 @@
+import { React } from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import { Menu, Typography, Button } from '@material-ui/core';
+import CloseIcon from '@material-ui/icons/Close';
+
+import useStyles from './headerStyles';
+import notificationsData from './dummyData';
+
+const Notifications = (props) => {
+  const classes = useStyles();
+
+  const notifAsign = (item) => (
+    <>
+      <Typography variant="body1">
+        {item.body}
+      </Typography>
+      <Typography variant="subtitle2" className={classes.bold}>
+        {item.level}
+      </Typography>
+      <Button
+        disableElevation
+        component={Link}
+        to='/test'
+        color="primary"
+        variant="text"
+        className={classes.seemoreBtn}
+        onClick={props.handleNotifClose}>
+          See More
+      </Button>
+    </>
+  );
+
+  const notifTest = (item) => (
```

Comment:
notificationTest isn't so long but easier to read

Reply:
https://github.com/mtalstykh-exadel/team4_frontend/blob/61508cbbb6209a5b4971fa235ad9991b4978da1a/src/components/header/notificationsMenu.js#L29-L34